### ISSUE TITLE
Check functionality on Edge

### ISSUE THREAD

**Issue by vjrantal** [vjrantal/badge-builder#19]

**Comment by deadlyfingers** [comment]
Well it should work - 
https://www.polymer-project.org/1.0/docs/browsers
I'm going to tinker with the index.html and see if changing the layout will help as even the hacky redraw I added doesn't cut it

**Comment by deadlyfingers** [comment]
I noticed that we don't have the Edge meta tag - so maybe this needs to be added to the index.html head?
```
<meta http-equiv="X-UA-Compatible" content="IE=Edge">
```

**Comment by vjrantal** [comment]
@deadlyfingers: I don't think that would fix the issue we were seeing. There are also some reasons not to use it http://stackoverflow.com/a/26348511 .

**Comment by deadlyfingers** [comment]
ok, my other thought is I noticed that there were a number of issues with the recent iron-list build so maybe we use an older version that didn't have so many new developments, or check for an updated build with bower

**Comment by deadlyfingers** [comment]
IE/Edge won't show saved badges in iron-list even after updating it to the latest version. No current issues mention it so will need further investigation -
